app/api/thesauri/thesauri.js

```diff
@@ -86,7 +86,10 @@ async function updateOptionsInEntities(current, thesauri) {
 
 const update = async thesauri => {
   const currentThesauri = await model.getById(thesauri._id);
-  if (JSON.stringify(thesauri.values) !== JSON.stringify(currentThesauri.values)) {
+  const valuesHaveChanged =
+    JSON.stringify(thesauri.values) !== JSON.stringify(currentThesauri.values);
+  const nameHasChanged = thesauri.name !== currentThesauri.name;
+  if (valuesHaveChanged || nameHasChanged) {
```

Comment:
@mfacar pleas change this to not use JSON.stringify.

Reply:
Currently, when the order of the thesaurus options has changed, we should update the model because this order of options is taken into account in selectors for example. In this case, it could be a cheap option to keep the comparison with the current approach.